config/samples/orchestrator_v1alpha1_orchestrator.yaml

```diff
@@ -40,19 +40,19 @@ spec:
       installPlanApproval: Automatic # whether the update should be installed automatically
       name: rhdh # name of the operator package
   rhdhPlugins: # RHDH plugins required for the Orchestrator
-    npmRegistry: ""
-    scope: "@janus-idp"
+    npmRegistry: "" # set npm registry. If left empty, defaults to https://registry.npmjs.org. Can be set to https://npm.stage.registry.redhat.com
+    scope: "" # set appropriate scope based on npm registry set above. Can be set to @redhat if using registry https://npm.stage.registry.redhat.com above.
```

Comment:
@masayag should we have a default value to redhat for OCP?

Reply:
We should prefer using the @redhat scope from the `https://npm.registry.redhat.com/` registry, but as for now, we don't have the plugins for 1.2 available there, so we'll have to use for now the `@janus-idp` for that (https://registry.npmjs.com/@janus-idp) 

But as soon as there will be a first release available, we should use the redhat's one.

Reply:
ack, should we then default to @janus-idp for now?

Reply:
yes.

Reply:
taking my yes back for a minute:
there isn't a good answer for that. the current values are:
```
  npmRegistry: "https://npm.stage.registry.redhat.com" # NPM registry is defined already in the container, but sometimes the registry need to be modified to use different versions of the plugin, for example: staging(https://npm.stage.registry.redhat.com) or development repositories
  scope: "@redhat"
```
This registry cannot be accessed externally, therefore we don't have a way for now to support 1.2 installation.
so, it doesn't matter. it can remain as is or be changed. it  won't work in both cases. :-(

Reply:
you lost me at `it won't work in both cases :-(` 😁 . So neither `janus-idp` or `redhat` won't work? Is it something we should be concerned about? or maybe it's not a big deal....

Reply:
For now, I updated registry and scope to default to `https://registry.npmjs.com` and `@janus-idp` with comment indicating the user can update it as needed.

Reply:
@jenniferubah after discussing with @masayag, I think it would be best to leave it empty as suggested. Sorry for the confusion 😄

Reply:
No worries at all. I've updated it to empty now :grinning: